samples-v2/function_chaining/tests/test_say_hello.py

```diff
@@ -0,0 +1,4 @@
+# Activity functions require no special implementation aside from standard Azure Functions
+#   unit testing for Python. As such, no test is implemented here.
+# For more information about testing Azure Functions in Python, see the official documentation:
+# https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-python#unit-testing
```

Comment:
Why are these `test_{activity_name}.py` files all empty?

Reply:
Thought the comment explained this. I didn't find it necessary to actually implement tests for the activities, as they are no different than any other Python function. I can, though, if that helps with clarity